tests/resources/sidecar_fit.xml

```diff
@@ -63,6 +63,7 @@
       <Topstuk_ID>topstuk15</Topstuk_ID>
       <Word_ID>word14</Word_ID>
       <WorkPID>pid16</WorkPID>
+      <MEEMOO-PID>7m03z1634f</MEEMOO-PID>
```

Comment:
De aangeleverde meemoo PID wordt dan ook als local id in MH opgeslagen. Op zich niet erg, en geeft een tracée dat er een meemoo PID is meegeleverd in de SIP.